src/app/[locale]/projects/detail/[id]/components/Obligations/ObligationsView/LicenseObligation.tsx

```diff
@@ -0,0 +1,210 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { useTranslations } from 'next-intl'
+import Link from 'next/link'
+import { useState, useEffect } from 'react'
+import { Table, _ } from '@/components/sw360'
+import { ProjectObligationsList, Session, LicenseObligationRelease } from '@/object-types'
+import { useSession } from 'next-auth/react'
+import { BsCaretDownFill, BsCaretRightFill } from 'react-icons/bs'
+import { Spinner } from 'react-bootstrap'
+import { SW360_API_URL } from '@/utils/env'
+
+const Capitalize = (text: string) =>
+    text.split('_').reduce((s, c) => s + ' ' + (c.charAt(0) + c.substring(1).toLocaleLowerCase()), '')
+
+function ExpandableList({ previewString, releases }: { previewString: string, releases: LicenseObligationRelease[] }) {
+    const [ isExpanded, setExpanded ] = useState(false)
+    return (
+        <>
+            {
+                isExpanded ?
+                <div>
+                    <span><BsCaretDownFill onClick={() => setExpanded(false)} />{' '}</span>
+                    {releases.map((release: LicenseObligationRelease, index: number) => {
+                        return (
+                            <li key={release.id ?? ''} style={{ display: 'inline' }}>
+                                <Link href={`/components/releases/detail/${release.id ?? ''}`} className='text-link'>
+                                    {`${release.name ?? ''} ${release.version ?? ''}`}
+                                </Link>
+                                {index >= releases.length - 1 ? '' : ', '}{' '}
+                            </li>
+                        )
+                    })}
+                </div>:
+                <div>
+                    {
+                        releases.length !== 0 &&
+                        <div><BsCaretRightFill onClick={() => setExpanded(true)} />{' '}{previewString}</div>
+                    }
+                </div>
+            }
+        </>
+    )
+}
+
+function ShowObligationTextOnExpand({id, infoText, colLength}: {id: string, infoText: string, colLength: number }): JSX.Element {
+    const [isExpanded, setIsExpanded] = useState(false)
+    useEffect(() => {
+        if(isExpanded) {
+            const el = document.getElementById(id)
+            const par = el.parentElement.parentElement.parentElement
+            console.log(par)
+            const tr = document.createElement('tr')
+            tr.id = `${id}_text`
+            const td = document.createElement('td')
+            td.colSpan = colLength
+            const licenseObligationText = document.createElement('p')
+            licenseObligationText.style.whiteSpace = "pre-line"
```

Comment:
replace double quote by single quote
```suggestion
            licenseObligationText.style.whiteSpace = 'pre-line'
```